Sources/URBNSwiftyConvenience/UIView+Additions.swift

```diff
@@ -158,4 +158,35 @@ extension UIView {
         // Alert the layout needs updating
         setNeedsLayout()
     }
+    
+    func sizeToWidthOfContainer(){
+        sizeToWidthOfContainerWithEqualInset(0)
+    }
+    
+    func sizeToWidthOfContainerWithEqualInset(_ inset: CGFloat){
```

Comment:
@BryanDeemer thoughts on this naming?
`func sizeToWidthOfContainer(withEqualInset inset: CGFloat)`

Reply:
That's fine with me. I'll change it now